### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by steventhanna** [sotch-pr35mac/skrum#7]
The system says that `Cannot read length of undefined`, when not ten minutes ago, with no changes even made, it worked perfectly fine... Why... 
```html
<div class="card-body">
  Owner:
  <strong><%= user.projects[i].owner %></strong>
  Users:
  <% for(var a = 0; a <user.projects[i].userList.length; a++) { %>
    <%= user.projects[i].userList[a] %> 
  <% } %>
</div>
```

More specifically, this line:
```html
<% for(var a = 0; a <user.projects[i].userList.length; a++) { %>
```
@sotch-pr35mac: you seem to know this shit better than me... Ideas?

**Comment by sotch-pr35mac** [comment]
i will take more of a look into this when i get home, thank you for pointing it out.

**Comment by steventhanna** [comment]
I think I figured out what the problem is... Now, since I was able to get the add project button to work correctly, there is actually data in the database.  And I just realized that projects are not being passed through the controller, to the page.  

I started an implementation, but I feel there might have been an error along the way.  Could @sotch-pr35mac  take a look at it for me?  It is in the `UserController.js` file.  Thanks much
